packages/odyssey-react-mui/src/Button.tsx

```diff
@@ -168,6 +171,7 @@ const Button = ({
           ref={localButtonRef}
           size={size}
           startIcon={startIcon}
+          tabIndex={tabIndex}
```

Comment:
This may not be an issue, but what if `tabIndex` is undefined, since it's optional? One way to only add this prop to the component would be something like this:
```
startIcon={startIcon}
{...(tabIndex !== undefined && { tabIndex })}
translate={translate}
```

This way, we only include tabIndex if we meant to.

Reply:
If undefined, it assigns tabindex a value of 0, which, is the default value